Sources/DockProgress/util.swift

```diff
@@ -106,3 +110,31 @@ extension NSBezierPath {
 		self.init(roundedRect: rect, xRadius: cornerRadius, yRadius: cornerRadius)
 	}
 }
+
+final class VerticallyCenteredTextLayer: CATextLayer {
+
+	convenience init(frame rect: CGRect, center: CGPoint) {
+		self.init()
+		frame = rect
+		frame.center = center
+		alignmentMode = .center
+		truncationMode = .end
+		font = NSFont(name: "HelveticaNeue-Bold", size: 0)
+		fontSize = 25
+		contentsScale = NSScreen.main!.backingScaleFactor
```

Comment:
```suggestion
		contentsScale = NSScreen.main?.backingScaleFactor ?? 2
```

The current code will fail if the system has no screen.